### DISCUSSION TITLE
[Feature Request] "Add to Collection" in Compare window (and custom compare strings)

### DISCUSSION THREAD

**Discussion by KAndrsn** [character-map-uwp/Character-Map-UWP#223] [Ideas]
1.  Add a right-click menu to the font 'cards' in the compare window, so the "Add to Collection" is available.
2. Add some way to edit the stings in the drop down list.  (e.g. I wanted to see "|1IliL 0Oo pgq" and I had to retype it because I accidentally changed the drop down selection.)

**Comment by JohnnyWestlake** [comment]
Thanks for the request - number 1 I'll add to the next build.

Number 2 I'll give some thought for a future build - no guarantee it'll happen from my end, but it is something I've considered before.

**Reply by JohnnyWestlake** [reply]
@KAndrsn So the first part of this should now be in the store builds.

**Comment by KAndrsn** [comment]
Sorry for the late reply.  Thank you.  It makes collection building much easier, IMO.

**Comment by JohnnyWestlake** [comment]
Number 2 was also implemented some time back in Settings -> Preview Suggestions. Enjoy!
